### PR TITLE
New CHANGELOG file

### LINKED ISSUE #1727: Add a CHANGELOG.md file to this repo
I'd like to propose adding a monthly "changelog" to this repository to detail each commit that lands in the `whosonfirst-data-*` repositories.

Since the administrative data has split into ~260 per-country repositories, it can be challenging to track each change. Adding a changelog with brief commit/PR descriptions would shed a bit of light on what updates have been done each month.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1727 

This PR adds a new `CHANGELOG` to this repo, with links and descriptions that outline the work done to Who's On First admin records in Nov 2019.

I'm open to formatting changes, but the general contents should include relevant work for the month.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,62 @@
+November 2019 - Who's On First Updates
+
+- Updated locality and neighbourhood records in Norway (issue: #298)
```
Comment (nvkelso) [inline CHANGELOG]: Please prefix this `NORWAY: `

File: CHANGELOG [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - Resetting all `lbl:bbox` values for records that received new Polygon or 
+      MultiPolygon geometries
+    - Validating all geometries using osgeo, validating all records using Who's On First's
+      `go-whosonfirst-validate` tool
+    - Adding Wikipedia and Wikidata-sourced name translations to any record without a name
+      translation
+    - Completing PIP work to updating or confirming all `wof:hierarchy` properties for all
+      records in the Poland admin repository
+
+- Corrected postalcode hierarchies in France (issue: #1713)
```
Comment (nvkelso) [inline CHANGELOG]: Please prefix this section with `FRANCE: `:

File: CHANGELOG [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  Fixed by: https://github.com/whosonfirst-data/whosonfirst-data-postalcode-fr/pull/2
+    - Updating "county_id" and "localadmin_id" values in dozens of postalcode records'
+      `wof:hierarchy` properties, also updating `wof:belongsto` and `wof:parent_id` values
+    - New postalcode hierarchies in France now maintain the appropriate, current parent
+      administrative records
+
+- Addition of `src:alt_label` property to each alt file (issue: #1714)
```
Comment (nvkelso) [inline CHANGELOG]: Please prefix this with `ALTERNATE GEOMETRIES: `

File: CHANGELOG [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  Fixed by: Multiple, example: 
+  https://github.com/whosonfirst-data/whosonfirst-data-admin-ro/pull/9
+    - In order for Who's On First to property publish public SQLite distribution files,
+      each "alt" file in Who's On First needed a `src:alt_label` property added.
+    - Alt files in each of the 260 per-country Who's On First repositories were given this
+      property in a series of pull requests.
+
+- Updated county, localadmin, locality, borough, and neighbourhood records in
```
Comment (nvkelso) [inline CHANGELOG]: Please prefix with `POLAND: `:

File: CHANGELOG [hunk 5]
```diff
@@ -0,0 +1,62 @@
+November 2019 - Who's On First Updates
```
Comment (nvkelso) [inline CHANGELOG]: # Who's On First Changelog

_This is a human generated overview of significant changes to Who's On First place data across all repos. More recent changes are at the top, oldest changes at the bottom._

## 2019 November

File: CHANGELOG [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+      property, as necessary
+    - Demoting some locality records to the neighbourhood placetype, as needed
+    - Adding new borough records to two localities in Poland - Krakow and Warsaw
+    - Adding concordances to the `wof:concordances` propert
+    - Validating all geometries using osgeo, validating all records using Who's On First's
+      `go-whosonfirst-validate` tool
+    - Adding Wikipedia and Wikidata-sourced name translations to any record without a name
+      translation
+    - Completing PIP work to updating or confirming all `wof:hierarchy` properties for all
+      records in the Poland admin repository
```
Comment (nvkelso) [inline CHANGELOG]: Please add a note at the bottom saying:

> _NOTE: This document was created 2019 November. Earlier changes have not been retroactively cataloged._

**Comment by nvkelso** [comment]
(You'll also need to add some whitespace lines for new Markdown formatting.)

**Comment by stepps00** [comment]
The most recent commit fixes formatting, changes filename, adds header and footer, and adds sub-headers.

Back to @nvkelso for :eyes:

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+# Who's On First Changelog
+_This is a human generated overview of significant changes to Who's On First place data 
+across all repos. More recent changes are at the top, oldest changes at the bottom._
+
+## 2019 October
+
+### NORWAY:
+- Updated locality and neighbourhood records in Norway (issue: #298)
```
Comment (nvkelso) [inline CHANGELOG.md]: Nit, here and elsewhere:

Please add specific link to the Issue, eg:

Issue [#298](https://github.com/whosonfirst-data/whosonfirst-data/issues/298)

File: CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  - Updating "county_id" and "localadmin_id" values in dozens of postalcode records' `wof:hierarchy` properties, also updating `wof:belongsto` and `wof:parent_id` values
+  - New postalcode hierarchies in France now maintain the appropriate, current parent administrative records
+
+### ALTERNATE GEOMETRIES:
+- Addition of `src:alt_label` property to each alt file (issue: #1714)
+- Fixed by: Multiple, example: https://github.com/whosonfirst-data/whosonfirst-data-admin-ro/pull/9
+  - In order for Who's On First to property publish public SQLite distribution files, each "alt" file in Who's On First needed a `src:alt_label` property added.
+  - Alt files in each of the 260 per-country Who's On First repositories were given this property in a series of pull requests.
+
+### POLAND:
```
Comment (nvkelso) [inline CHANGELOG.md]: Nit: don't need the `:` when in a separate headline line like this (and elsewhere)

**Comment by stepps00** [comment]
Most recent commit removes `:` in sub-headers and fixes issue links.. merging.

**Pull request merged**